scripts/release-on-comment.py

```diff
@@ -211,5 +230,13 @@ def main() -> None:
     trigger_release(Path(options.payload), options.token)
 
 
+def run_capturing(command):
```

Comment:
I believe this wrapper is not necessary:

- Instead of doing the decoding ourselves, can pass `text=True` to `run` and then `completed_process.stdout` will be in text-mode.
- The `print` I think can be removed, because `print_and_exit()` already prints the message.
- Checking the `returncode` can be replaced by passing `check=True` to `run`.
- The output capturing can be replaced with adding `capture_output=True`  to `run`. The output will be included in the `CalledProcessError` raised by `check=True` as well.

Reply:
> The print I think can be removed, because print_and_exit() already prints the message.

`print_and_exit()` will only print the exception, not the entire captured output, which is useful to have.

> The output capturing can be replaced with adding capture_output=True to run. The output will be included in the CalledProcessError raised by check=True as well.

Oh awesome, that I didn't know. Indeed then this wrapper is not really necessary, I agree.

Reply:
We also need to then change the call of `print_and_exit` to include `stdout` and `stderr` in that case.

Reply:
Now that we've removed the wrapper. How do we use `completed_process.stdout` or `completed_process.stderr` ?
We'd have to store the last completed_process and use that?

Reply:
Sorry, I don't really remember what `completed_process` was. Can you rephrase the question in terms of the current state of the PR?